docs/getting-started/quickstart-programming.md

```diff
@@ -53,13 +53,56 @@ In this case, `Greeting` is a static member of the `Greetings` class.
 
 To learn more about HTTP serving in DBOS, see [our guide](../tutorials/http-serving-tutorial).
 
+### Creating Database Tables
+
+:::info what you will learn
+How to create and manage database tables.
+:::
+
+Let's create a database table our app can use to store greetings.
+In DBOS, we recommend managing database tables using [schema migrations](https://en.wikipedia.org/wiki/Schema_migration).
+By default, we use [Knex](https://knexjs.org/) to manage migrations, but also support other tools including [TypeORM](https://typeorm.io/) and [Prisma](https://www.prisma.io/).
+To create a new migration file, run the following command:
+
+```
+npx knex migrate:make greetings
```

Comment:
Should we remove `dbos_greetings` from our template? This is no longer used anywhere.